pkg/apis/elasticsearch/v1/name.go

```diff
@@ -174,3 +175,7 @@ func RemoteCaSecretName(esName string) string {
 func FileSettingsSecretName(esName string) string {
 	return ESNamer.Suffix(esName, fileSettingsSecretSuffix)
 }
+
+func StackConfigElasticsearchConfigSecretName(esName string) string {
+	return ESNamer.Suffix(esName, policyEsConfigSecretSuffix)
+}
```

Comment:
We need to prevent conflicts with other existing secrets managed by ECK when copying additional secrets referenced in `spec.elasticsearch.secretMounts` of a StackConfigPolicy.

Example: esName: xyzb, nodeSetName: default => Secret `xyzb-es-default-es-config` in conflict with Secret `default-es-config` if referenced in secretMounts.

```suggestion
}

func StackConfigAdditionalSecretName(esName string, secretName string) string {
	return ESNamer.Suffix(esName, "scp", secretName)
}
```